templates/sst/default/templates.yaml

```diff
@@ -0,0 +1,35 @@
+dataset: sst
+subset: default
+templates:
+  5119a0b5-5d82-4401-900a-7fafc1d48ff6: !Template
+    id: 5119a0b5-5d82-4401-900a-7fafc1d48ff6
+    jinja: 'How positive is the movie review below?
+
+      Give a score on a scale from 0 to 1.
+
+
+      {{sentence}} |||
+
+      {{label}}'
+    name: sentiment_scoring
+    reference: ''
+  b15994be-ca57-4924-9af7-fbaa6ee0124b: !Template
+    id: b15994be-ca57-4924-9af7-fbaa6ee0124b
+    jinja: 'Is the movie review below positive?
+
+
+      {{sentence}} |||
+
+      {{["no","yes"]
+
+      [0 if label < 0.5 else 1]
+
+      }}'
+    name: sentiment_classification
+    reference: ''
+  c5dc88f9-204f-43be-b09f-f6334215e271: !Template
+    id: c5dc88f9-204f-43be-b09f-f6334215e271
+    jinja: "Form a sentence parse tree (formatted as a parent pointer tree) using\
```

Comment:
Can you add spaces in this parse? It seems very hard for a language model to get this.